app/Models/NewsPost.php

```diff
@@ -170,21 +173,21 @@ public static function syncAll()
         }
     }
 
-    public function scopeDefault($query)
+    public function scopeDefault(Builder $query): Builder
```

Comment:
apparently scopes are supposed to return void (or more like the return isn't used)